src/test/java/UploadReceiptServletTest.java

```diff
@@ -56,6 +56,7 @@
 import java.util.HashMap;
```

Comment:
I accidentally merged #75 too early, so most of this file has already been reviewed.